scripts/version_bump.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"strconv"
+)
+
+const versionFile = "../internal/version/version.go"
+
+func main() {
+	var bumpType string
+	if len(os.Args) > 1 {
```

Comment:
if we use the arg approach above, we may want this to be a flag or something to differentiate. Up to you though

Reply:
Done! Added an extra parameter ```-bump``` which takes the three bump types ```minor```, ```major``` and ```patch```